contracts/src/test/ScrollChain.t.sol

```diff
@@ -563,18 +617,24 @@ contract ScrollChainTest is DSTestPlus {
         rollup.removeSequencer(_sequencer);
         hevm.stopPrank();
 
-        // change to random operator
-        hevm.expectEmit(true, false, false, true);
-        emit UpdateSequencer(_sequencer, true);
-
-        assertBoolEq(rollup.isSequencer(_sequencer), false);
-        rollup.addSequencer(_sequencer);
-        assertBoolEq(rollup.isSequencer(_sequencer), true);
-
-        hevm.expectEmit(true, false, false, true);
-        emit UpdateSequencer(_sequencer, false);
-        rollup.removeSequencer(_sequencer);
-        assertBoolEq(rollup.isSequencer(_sequencer), false);
+        // revert when account is not EOA
+        if (_sequencer.code.length > 0) {
+            hevm.expectRevert("not EOA");
```

Comment:
could we make sure this branch will be hit?

Reply:
based on my experience, it will always be hit in the fuzzing tests. Anyway, I add a test for it.